### PR TITLE
Fix aleph sequental reader source has no f001

### PR DESCRIPTION
**Pull request by natlibfi-jonollil**
Reader checks if 001 field was in alephsequental and adds it if needed

### CODE REVIEW

File: test-fixtures/aleph-sequential/noF001 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+000000004 FMT   L BK
+000000004 LDR   L
```
Comment (hmakila) [inline test-fixtures/aleph-sequential/noF001:2]: LDR missing

File: test-fixtures/aleph-sequential/yesF001 [hunk 2]
```diff
@@ -0,0 +1,25 @@
+LDR
```
Comment (hmakila) [inline test-fixtures/aleph-sequential/yesF001:1]: This record is not in AlephSeq format

Comment (natlibfi-jonollil) [reply test-fixtures/aleph-sequential/yesF001]: It is marc-json-js .toString() format. (https://github.com/NatLibFi/marc-record-js/blob/3694d70041463e415fa9bbb473fb441cd2845081/src/index.js#L165-L179)

File: test-fixtures/aleph-sequential/noF001 (hunk 1 above)
Comment (natlibfi-jonollil) [reply test-fixtures/aleph-sequential/noF001]: Added missing dummy leader

**Review by hmakila — APPROVED** [review]

**Review by ammsalme — APPROVED** [review]
Questions: why are we forcing records from Aleph Sequential to have f001? Do other formats force f001?

**Review by ammsalme — APPROVED** [review]

**Pull request merged**